internal/daemon/controller/handlers/sessions/session_service.go

```diff
@@ -164,51 +205,74 @@ func (s Service) ListSessions(ctx context.Context, req *pbs.ListSessionsRequest)
 		return nil, errors.Wrap(ctx, err, op)
 	}
 
-	sesList, err := repo.ListSessions(ctx, session.WithTerminated(req.GetIncludeTerminated()), session.WithLimit(-1))
-	if err != nil {
-		return nil, err
-	}
-	if len(sesList) == 0 {
-		return &pbs.ListSessionsResponse{}, nil
-	}
-
-	filter, err := handlers.NewFilter(ctx, req.GetFilter())
-	if err != nil {
-		return nil, err
-	}
-	finalItems := make([]*pb.Session, 0, len(sesList))
-	res := perms.Resource{
-		Type: resource.Session,
-	}
-	for _, item := range sesList {
-		res.Id = item.GetPublicId()
-		res.ScopeId = item.GetProjectId()
-		authorizedActions := authResults.FetchActionSetForId(ctx, item.GetPublicId(), IdActions, auth.WithResource(&res)).Strings()
-		if len(authorizedActions) == 0 {
-			continue
+	var listResp *pagination.ListResponse[*session.Session]
+	sortBy := "created_time"
```

Comment:
We changed this in the target PR to set in each if statement explicitly, do you mind making that change here?